test/scripts/install_chromedriver_travis.sh

```diff
@@ -13,11 +14,17 @@ echo "Uninstalling current Google Chrome from `which google-chrome`..."
 sudo apt-get purge google-chrome-stable
 rm ~/.config/google-chrome/ -rf
 
-echo "Installing $CHROME_VERSION..."
-sudo wget --no-verbose -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
-echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list
-sudo apt-get update -qqy
-sudo apt-get -qqy install ${CHROME_VERSION}
+# echo "Installing $CHROME_VERSION..."
```

Comment:
Why do you want to commit commented out code? Please add a comment to explain why it is commented out